scripts/webpack.conf.js

```diff
@@ -1,47 +1,76 @@
 const { modifyWebpackConfig, shallowMerge, defaultOptions } = require('@gera2ld/plaid');
+const { isProd } = require('@gera2ld/plaid/util');
 const webpack = require('webpack');
 const WrapperWebpackPlugin = require('wrapper-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 const projectConfig = require('./plaid.conf');
 
 const INIT_FUNC_NAME = 'VMInitInjection';
+const definitions = new webpack.DefinePlugin({
+  'process.env.INIT_FUNC_NAME': JSON.stringify(INIT_FUNC_NAME),
+  'process.env.DEBUG': JSON.stringify(process.env.DEBUG || false),
+});
 
 module.exports = Promise.all([
-  modifyWebpackConfig(async (config) => {
-    config.plugins.push(
-      new webpack.DefinePlugin({
-        'process.env.INIT_FUNC_NAME': JSON.stringify(INIT_FUNC_NAME),
-        'process.env.DEBUG': JSON.stringify(process.env.DEBUG || false),
-      }),
-    );
-    return config;
-  }),
-  modifyWebpackConfig(async (config) => {
+  modify(),
+  modify((config) => {
     config.output.libraryTarget = 'commonjs2';
     config.plugins.push(
       new WrapperWebpackPlugin({
         header: `\
-window.${INIT_FUNC_NAME} = function () {
-  var module = { exports: {} };
-`,
+          window.${INIT_FUNC_NAME} = function () {
+            var module = { exports: {} };
+          `,
         footer: `
-  var exports = module.exports;
-  return exports.__esModule ? exports['default'] : exports;
-};0;`,
+            var exports = module.exports;
+            return exports.__esModule ? exports['default'] : exports;
+          };0;`,
       }),
     );
-    return config;
   }, {
+    'injected-web': {
+      entry: './src/injected/web',
+    },
+  }),
+  modify(null, {
+    injected: {
+      entry: './src/injected',
+    },
+  }),
```

Comment:
Why is `injected` splitted into a new webpack config?

Reply:
Since I don't know how to apply custom Terser settings by file mask, I've moved `injected` here.